provider/azure/storage.go

```diff
@@ -607,6 +607,8 @@ func (v *azureVolumeSource) AttachVolumes(ctx context.ProviderCallContext, attac
 	return results, nil
 }
 
+const azureDiskDeviceLink = "/dev/disk/azure/scsi1/lun%d"
```

Comment:
is it always scsi1?

Reply:
From what I can tell from the docs and examples online, yeah